pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTPrimaryExpression.java

```diff
@@ -11,6 +11,7 @@
  * <pre class="grammar">
  *
  * PrimaryExpression ::= {@linkplain ASTLiteral Literal}
+ *                     | {@link ASTClassLiteral ClassLiteral}
```

Comment:
I'll change that to "linkplain" for consistency :)

Reply:
Ok, for rendering, this doesn't make a difference, since the whole block is in `<pre`...